doc/toolbox-init-container.1.md

```diff
@@ -20,13 +13,55 @@ be used as the entry point for all toolbox containers, and must be run inside
 the container that's to be initialized. It is not expected to be directly
 invoked by humans, and cannot be used on the host.
 
+Entry point of a container can not be changed which makes container a bit hard
+to control. But it is possible to change the functionality of the command that
+is set for a container. This is what `toolbox init-container` is for.
+
+When a toolbox container is started for the first time, `toolbox
+init-container(1)` will create a user according to the options `--home`,
+`--shell`, `--uid` and `--user`. The user information can later be changed
+manually using tools like `usermod(1)`.
+
+On some systems some important paths like `/home`, `/media` or `/mnt` are
+symbolic links to different locations. Toolbox detects such scenarios and
+matches such configurations in the container. This behaviour is controlled using
+options `--home-link`, `--media-link` and `--mnt-link`.
+
+Every time a toolbox container is started, `toolbox init-container` does the
+following:
+
+- sets `XDG_RUNTIME_DIR` if it is unset
```

Comment:
I wonder if this is straying too far into documenting the details of the implementation. :)

It's definitely a thin line, and a matter of finding some balance between (a) what's actually useful for users of Toolbox versus contributors to Toolbox, and (b) what's meant to be a somewhat stable user-facing characteristic of a Toolbox container versus what's merely an implementation detail and can change over time.